Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1
 
 # https://github.com/michaelosthege/pythonnet-docker
-FROM --platform=linux/amd64 mosthege/pythonnet:python3.9.16-mono6.12-pythonnet3.0.1
+FROM --platform=linux/amd64 mosthege/pythonnet:python3.10.10-mono6.12-pythonnet3.0.1
```

Comment:
I think we support 3.9-3.12

Reply:
well, I think this is no longer true
[af3956f9](https://github.com/MannLabs/alphadia/pull/239/commits/af3956f9b14f6e15c1bdd6ef7252be47b4096024)

Reply:
Is it because type hint of union and tuple
In alphabase, it is Union[int, str, float] and Tuple[int, str, float].
In alphadia, it is int | str | float, and (int, str, float).